lib/tds.js

```diff
@@ -418,6 +450,8 @@ function finalizeDNRRulesAndLookup (startingRuleId, dnrRules) {
  *   The generated Tracker Blocking declarativeNetRequest ruleset.
  * @property {object} matchDetailsByRuleId
  *   Rule ID -> match details.
+ * @property {object} customActionRules
```

Comment:
Please could you rename this `inverseRulesByCustomRuleAction` and expand the comment (maybe "List of inverse rules to add for each custom rule action"?).

Reply:
Sure done, though to be pedantic I called it `inverseCustomActionRules`, since its an array not a map. :)